lib/led/array.js

```diff
@@ -0,0 +1,79 @@
+module.exports = function(Led, priv) {
+
+  /**
+   * LedArray()
+   * new LedArray()
+   *
+   * Create an Array-like object instance of Leds
+   * @alias Led.Array
+   * @return {LedArray}
+   */
+  var LedArray = function(numsOrObjects) {
+    if (!(this instanceof LedArray)) {
+      return new LedArray(numsOrObjects);
+    }
+
+    var pins = [];
+
+    if (numsOrObjects) {
+      while (numsOrObjects.length) {
+        var numOrObject = numsOrObjects.shift();
+        if (!(numOrObject instanceof Led)) {
+          numOrObject = new Led(numOrObject);
+        }
+        pins.push(numOrObject);
+      }
+    } else {
+      pins = pins.concat(Array.from(priv.keys()));
```

Comment:
I'm willing to give this up. We'll just document it as a breaking change—sound good?